cmd/crc/cmd/status.go

```diff
@@ -79,25 +79,41 @@ func runWatchStatus(writer io.Writer, client *daemonclient.Client, cacheDir stri
 		ramBar  *pb.ProgressBar
 	)
 
+	var err error
+	barPull = pb.NewPool()
 	isPullInit := false
-
-	err := client.SSEClient.Status(func(loadResult *types.ClusterLoadResult) {
+	err = barPull.Start()
+	if err != nil {
+		return err
+	}
+	defer func() {
+		err = barPull.Stop()
+	}()
+	err = client.SSEClient.Status(func(loadResult *types.ClusterLoadResult) {
 		if !isPullInit {
 			ramBar, cpuBars = createBars(loadResult.CPUUse, writer)
-			barPull = pb.NewPool(append([]*pb.ProgressBar{ramBar}, cpuBars...)...)
-			isPullInit = true
-			err := barPull.Start()
-			if err != nil {
+			if stopErr := barPull.Stop(); stopErr != nil {
+				return
+			}
+			barPull.Add(append([]*pb.ProgressBar{ramBar}, cpuBars...)...)
+			if startErr := barPull.Start(); startErr != nil {
 				return
 			}
+			isPullInit = true
 		} else if len(loadResult.CPUUse) > len(cpuBars) {
 			newCPUCount := len(loadResult.CPUUse) - len(cpuBars)
 			oldCPUCount := len(cpuBars)
+			if stopErr := barPull.Stop(); stopErr != nil {
+				return
+			}
 			for i := 0; i < newCPUCount; i++ {
 				bar := createCPUBar(oldCPUCount+i, writer)
 				barPull.Add(bar)
 				cpuBars = append(cpuBars, bar)
 			}
+			if startErr := barPull.Start(); startErr != nil {
+				return
+			}
```

Comment:
Ah, it looks like the problem is calling `barPull.Add()` to add a new bar while the pool is started?